### ISSUE TITLE
Icon Genie v2.3.0 profile generator incorrectly checking for hash in splashscreenColor hex value

### ISSUE THREAD

**Issue by taliator** [quasarframework/quasar#7441]
**Describe the bug**
Icon Genie profile generator incorrectly checking for hash in splashscreenColor hex value, and won't accept a hex value without a hash.

**Codepen/jsFiddle/Codesandbox (required)**
N/A

**To Reproduce**
1. Try to generate an IconGenie profile (replacing with relevant filenames):
    `icongenie profile -o foo -a spa,capacitor -i app-icon.png -b app-splashscreen.png --quality 2 --splashscreen-color 286DD7`
2. See error where matching pattern expects the hash:
    ![image](https://user-images.githubusercontent.com/1482924/87695532-9726a680-c75d-11ea-9595-4a55d0088376.png)
3. Now try to generate a profile with the hash:
    `icongenie profile -o foo -a spa,capacitor -i app-icon.png -b app-splashscreen.png --quality 2 --splashscreen-color #286DD7`
    ![image](https://user-images.githubusercontent.com/1482924/87695684-cccb8f80-c75d-11ea-9170-17de53a8fc1b.png)
5. Now escape the hash
    `icongenie profile -o foo -a spa,capacitor -i app-icon.png -b app-splashscreen.png --quality 2 --splashscreen-color \#286DD7`
6. Profile generates successfully, so try to generate icons:
    `icongenie generate -p ./icongenie-foo.json`
7. See error (obviously, since the profile should *not* include the hash)
    ![image](https://user-images.githubusercontent.com/1482924/87695922-203ddd80-c75e-11ea-9471-fe756958e17f.png)

**Expected behavior**
The profile should generate without the hash prepending the color (remove the hash from the pattern matching check).

**Screenshots**
See above.

**Platform (please complete the following information):**
OS: MacOS Catalina 0.15.5
Node: v14.2.0
NPM: 6.14.4

**Additional context**
N/A

**Comment by IlCallo** [comment]
Have you tried as `--splashscreen-color=286DD7` too?

**Comment by taliator** [comment]
@IlCallo I had not, but I just ran
`icongenie profile -o foo -a spa,capacitor -i app-icon.png -b app-splashscreen.png --quality 2 --splashscreen-color=286DD7`
 and it returns the same error, because the regex is still checking for a hash:

![image](https://user-images.githubusercontent.com/1482924/87784998-626e2a00-c805-11ea-8d35-f2ed649baf28.png)

**Comment by rstoenescu** [comment]
As per the documentation and CLI help command description, the --splashscreen-color (and all other color related command line params) must be stripped of the preceding '#' char. Otherwise, '#' is interpreted by your shell as "comment" and anything after it is discarded.

```
    --splashscreen-color  Prefill the params.splashscreenColor property;
                          Background color to use for the splashscreen generator;
                          Default (if no theme-color is specified): fff
                          The color must be in hex format (NOT hexa) without the leading
                          '#' character. Transparency not allowed.
                          Examples: 1976D2, eee
```

**Comment by taliator** [comment]
@rstoenescu Yes, as per my steps detailed above, I only attempted with a hash because the error message (after attempting to generate per the instructions) showed that the regex includes a hash.

FWIW, this is not a work stopper, I was obviously able to edit the file to add in the splashscreen color manually, I'm just letting you know that it's a bug.

**Comment by webnoob** [comment]
I'll be able to test / look into this over the weekend all being well. Had my eye on it but hadn't got to it yet :)

**Comment by webnoob** [comment]
@taliator Can you confirm that this is only happening when using the profile command please? So if you run `icongenie generate -a spa,capacitor -i app-icon.png -b app-splashscreen.png --quality 2 --splashscreen-color 286DD7` it works as expected?

**Comment by taliator** [comment]
@webnoob Ah indeed! Running `icon generate ... --splashscreen-color 286DD7` does **not** throw the error, whereas `icon profile ... --splashscreen-color 286DD7` does.

**Comment by webnoob** [comment]
Ok good, the fix will be made available after the next round of updates.

**Comment by webnoob** [comment]
Fixed in @quasar/icongenie-v2.3.1
